test/elixir/lib/couch.ex

```diff
@@ -179,13 +179,13 @@ defmodule Couch do
       Keyword.get(
         options,
         :timeout,
-        Application.get_env(:httpotion, :default_timeout, 5000)
+        Application.get_env(:httpotion, :default_timeout, 60_000)
```

Comment:
Would prefer to see these parameterized higher up in the file somewhere, magic numbers hidden in code always give me the twitches :)

Reply:
So, there are already multiple layers of configurable settings on top of this. Maybe it'd be better to figure out how to set the `:httpotion` `:default_timeout` in the app environment, or pass the `:timeout` in the `options`? I find that a little confusing for a simple setup like this because if you see the 5000 down here you'll have to go hunting for the config that overrides it.

Or are you saying it'd be better to have one more layer of indirection that leads back up to a constant at the top of the module?

I'll try a couple of things and see If something makes sense (in the next PR).